cli.js

```diff
@@ -128,7 +152,41 @@ program.command("check")
         if (options.headers) options.body = false;
         else if (options.body) options.headers = false;
         try {
-            const res = await api.check();
+            const res = await api.account.identity();
+            if (options.raw) console.log(JSON.stringify(res));
+            else {
+                if (res._res.status < 400) spinner.succeed(`${res._res.status}: ${res._res.statusText}`);
```

Comment:
You sure about that? I think we should use like:
```suggestion
                if (res._res.status.startsWith(2)) spinner.succeed(`${res._res.status}: ${res._res.statusText}`);
```
since it can be any 2XX response usually they are supposed to be when everything goes fine.

Reply:
My logic is that 100-300 are not errors.  This shows the request as failed only for 400-500

Reply:
Understandable then